pages/pricing.js

```diff
@@ -386,7 +386,7 @@ export default function Premium({ user }) {
                               <span>
                                 {feature.name}{" "}
                                 {typeof feature.tiers[tier.name] ===
-                                "string" ? (
+                                  "string" ? (
```

Comment:
also please remove that extra space that you have added before "string" and make it as it was before